tools/bazel/legacy-rnjs/.yarn/patches/@angular-bazel-https-c46f484fb8.patch

```diff
@@ -0,0 +1,87 @@
+diff --git a/src/ng_module/partial_compilation.bzl b/src/ng_module/partial_compilation.bzl
+index 5fd9a73d0118485c93cf39be7a2330ebe6c24243..c155cea264f524b2289917c48af3043870163c13 100644
+--- a/src/ng_module/partial_compilation.bzl
++++ b/src/ng_module/partial_compilation.bzl
+@@ -16,10 +16,10 @@ ng_partial_compilation_flag = rule(
+ )
+ 
+ def _partial_compilation_transition_impl(_settings, _attr):
+-    return {"@npm//@angular/bazel/src:partial_compilation": True}
++    return {"@rules_angular//src/ng_project/config:partial_compilation": True}
+ 
+ partial_compilation_transition = transition(
+     implementation = _partial_compilation_transition_impl,
+     inputs = [],
+-    outputs = ["@npm//@angular/bazel/src:partial_compilation"],
++    outputs = ["@rules_angular//src/ng_project/config:partial_compilation"],
+ )
+diff --git a/src/ng_package/packager.mjs b/src/ng_package/packager.mjs
+index 20604c595be4b5bb0fea0bae0aeb2c2a1f4b83eb..265c2cf00aacf29844bdd1154ba2400f40810bc0 100755
+--- a/src/ng_package/packager.mjs
++++ b/src/ng_package/packager.mjs
+@@ -8,7 +8,7 @@
+ import * as fs from 'fs';
+ import * as path from 'path';
+ import { globSync } from 'tinyglobby';
+-import { analyzeFileAndEnsureNoCrossImports } from './cross_entry_points_imports';
++import { analyzeFileAndEnsureNoCrossImports } from './cross_entry_points_imports.mjs';
+ /**
+  * List of known `package.json` fields which provide information about
+  * supported package formats and their associated entry paths.
+@@ -28,19 +28,19 @@ function main(args) {
+     const params = fs.readFileSync(paramFilePath, 'utf-8').split('\n').map(unquoteParameter);
+     const [
+     // Output directory for the npm package.
+-    outputDirExecPath, 
```

Comment:
It might not be worth changing since we are about to bring int a whole new ng_package toolset, but alot of these patch changes are just removing a single space, which makes it more brittle in theory, since the patch change has no actual effect.